src/dpp/guild.cpp

```diff
@@ -200,8 +203,12 @@ std::string guild_member::build_json(bool with_id) const {
 			j["roles"].push_back(std::to_string(role));
 		}
 	}
-	j["mute"] = is_muted();
-	j["deaf"] = is_deaf();
+
+	if (flags & gm_voice_action) {
+		j["mute"] = is_muted();
+		j["deaf"] = is_deaf();
```

Comment:
you should probably clear gm_voice_action here after setting the values

Reply:
Why would it be harmful?

Reply:
Think it would be actually the best idea to clear it there yeah

Reply:
Well then clear it in guild_edit_member

Reply:
Can't, the guild member is passed as const reference